resources.go

```diff
@@ -1090,8 +1105,35 @@ func Provider() tfbridge.ProviderInfo {
 			"aws_globalaccelerator_endpoint_group": {Tok: awsResource(globalacceleratorMod, "EndpointGroup")},
 			"aws_globalaccelerator_listener":       {Tok: awsResource(globalacceleratorMod, "Listener")},
 			// Glue
-			"aws_glue_catalog_database":       {Tok: awsResource(glueMod, "CatalogDatabase")},
-			"aws_glue_catalog_table":          {Tok: awsResource(glueMod, "CatalogTable")},
+			"aws_glue_catalog_database": {
+				Tok: awsResource(glueMod, "CatalogDatabase"),
+				Fields: map[string]*tfbridge.SchemaInfo{
+					"name": {
+						Default: &tfbridge.DefaultInfo{
+							From: tfbridge.FromName(tfbridge.AutoNameOptions{
+								Transform: func(name string) string {
+									newName := fmt.Sprintf("%s_%s", name, TransformWithRandomString(8))
+									return strings.ToLower(newName)
```

Comment:
Why do we need this Transform?  Can't we parameterize the AutoNamer to do this without custom code here?  (OR is there a feature we should add there to handle this)?

Reply:
This transform stops adding any random numbers to the ID. This isn't something we have needed before. My thoughts are, if this needs moved to be a feature of it's own in the AutoNaming logic, then we will do so